Maquetado/scss/styles.scss

```diff
@@ -0,0 +1,173 @@
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-title {
+  font-size: 24px;
+  justify-content: flex-end;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.techno-title {
+  font-size: 12px;
+}
+
+.type-tool {
+  font-size: 24px;
+  position: relative;
+  transform: translateX(100%);
+
+  &::before {
+    background-image: url('assets/triangle.svg');
+    background-repeat: no-repeat;
+    content: '';
+    height: 90px;
+    position: absolute;
+    transform: translateX(-50%);
+    top: -35px;
+    width: 100px;
+    z-index: -1;
+  }
+}
+
+.tools-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.container-techno {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 100px;
+  padding: 50px;
+}
+
+.techno {
+  align-items: center;
+  background-color: $color-green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
```

Comment:
I think the height should be auto since you want it to occupy the children total size